tutor/commands/k8s.py

```diff
@@ -296,13 +297,18 @@ def start(context: K8sContext, names: List[str]) -> None:
     names = names or ["all"]
     for name in names:
         if name == "all":
-            # Create volumes
-            kubectl_apply(
-                context.root,
-                "--wait",
-                "--selector",
-                "app.kubernetes.io/component=volume",
-            )
+            # Create volumes if exists
+            volumes_manifest = tutor_env.pathjoin(context.root, "k8s/volumes.yml")
+            volumes_content = pathlib.Path(volumes_manifest).read_text()
```

Comment:
any particular reason for using Path here instead of open()?

Reply:
Just for brevity. Should I reimplement it using `open()`?

Reply:
Tutor code is using open or io.open. It is better to keep the new code consistent with the existing pattern rather than having a new one. Thanks